### PR TITLE
Update the swagger with ACS prefix

### PR DESCRIPTION
**Pull request by vikrampraveen**
Update the swagger with ACS prefix and add a field to SMSDeliveryReportReceivedEventData

### CODE REVIEW

File: specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "paths": {},
   "definitions": {
-    "ChatMessageReceivedEventData": {
```
Comment (jhendrixMSFT) [inline specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json:10]: This is a breaking change in a stable swagger.  Why is this required?

Comment (vikrampraveen) [reply specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json:10]: I created this swagger on friday, missed adding the prefix as the guidelines. So wanted update it.

Comment (vikrampraveen) [reply specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json:10]: The service is yet to launch in public preview, it will launch next month.

Comment (JeffreyRichter) [reply specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json:10]: Then I don't think this swagger file should be in a stable directory yet - it sends the wrong message to customers.
